ingestion/src/metadata/workflow/application_output_handler.py

```diff
@@ -8,27 +8,14 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-
 """
-Module handles the output messages for applications
+Generic Workflow entrypoint to execute Applications
 """
-import time
-
-from metadata.utils.helpers import pretty_print_time_duration
-from metadata.utils.logger import ANSI, log_ansi_encoded_string
-from metadata.workflow.output_handler import print_workflow_summary
 
+from metadata.utils.deprecation import deprecated
+from metadata.workflow.base import BaseWorkflow
 
-def print_status(workflow: "ApplicationWorkflow") -> None:
-    """
-    Print the workflow results
-    """
-    print_workflow_summary(workflow)
 
-    if workflow.runner.get_status().source_start_time:
-        log_ansi_encoded_string(
-            color=ANSI.BRIGHT_CYAN,
-            bold=True,
-            message="Workflow finished in time: "
-            f"{pretty_print_time_duration(time.time()-workflow.runner.get_status().source_start_time)}",
-        )
+@deprecated(message="Use 'workflow.print_status()' instead.", release="1.8")
+def print_status(workflow: BaseWorkflow):
+    workflow.print_status()
```

Comment:
❤️

I think we could move the deprecation to 1.6 IMO. It gives people an entire release cycle to move their stuffs.